Packs/AbnormalSecurity/Integrations/AbnormalSecurityEventCollector/test_data/test_get_threat.json

```diff
@@ -0,0 +1,95 @@
+{
+    "123456789-1": {
+        "threatId": "123456789-1",
+        "messages": [
+            {
+                "threatId": "123456789-1",
+                "abxMessageId": 1,
+                "receivedTime": "2022-10-01T18:44:38Z"
+            },
```

Comment:
This is not returned because it later than 'now' but it will not be true in two months and so the test will fail. We need UT to be determinist and hence will not be dependent on time. You will need to use the freeze time decorator, you can see some examples in other integrations 
https://github.com/spulec/freezegun#decorator